src/net/sourceforge/kolmafia/MonsterData.java

```diff
@@ -1838,51 +1788,33 @@ public boolean willUsuallyMiss(final int defenseModifier) {
   }
 
   public void appendItemDrops(StringBuilder buffer) {
```

Comment:
please add some tests for this refactor (confirming the same behaviour). I think it's probably fine, but it's hard to be sure purely by reading.

Reply:
I knew I should do this and to my shame I chose not to

Reply:
Done!

Reply:
another good test would be velvet underground dweller, who only has bounty items.